packages/mdc-chips/_mixins.scss

```diff
@@ -159,3 +163,23 @@
   height: $size;
   font-size: $size;
 }
+
+@mixin mdc-chip-leading-icon-margin(
+  $top: $mdc-chip-leading-icon-margin-top,
+  $right: $mdc-chip-leading-icon-margin-right,
+  $bottom: $mdc-chip-leading-icon-margin-bottom,
+  $left: $mdc-chip-leading-icon-margin-left) {
+  .mdc-chip__icon.mdc-chip__icon--leading {
```

Comment:
the `.mdc-chip__icon` is overly specific in this instance. `mdc-chip__icon--leading` can only apply to `mdc-chip__icon`, so you don't need to specify it again.

We did some research offline and realize that sometimes we need to be overly specific about our CSS selectors for icon elements, because we want to override styles applied by `material-icons-extended`

However,`margin` is applied by `material-icons-extended`, so it does not apply in this case.

So remove `mdc-chip__icon`, pretty please, and re-run the screenshot tests.